chain/types/src/lib.rs

```diff
@@ -239,10 +262,12 @@ where
             token: instances.token.address().to_hopr_address(),
             channels: instances.channels.address().to_hopr_address(),
             announcements: instances.announcements.address().to_hopr_address(),
-            node_safe_registry: instances.safe_registry.address().to_hopr_address(),
-            ticket_price_oracle: instances.price_oracle.address().to_hopr_address(),
-            winning_probability_oracle: instances.win_prob_oracle.address().to_hopr_address(),
-            node_stake_v2_factory: instances.stake_factory.address().to_hopr_address(),
+            module_implementation: instances.module_implementation.address().to_hopr_address(),
+            node_safe_migration: instances.node_safe_migration.address().to_hopr_address(),
+            node_safe_registry: instances.node_safe_registry.address().to_hopr_address(),
+            ticket_price_oracle: instances.ticket_price_oracle.address().to_hopr_address(),
+            winning_probability_oracle: instances.winning_probability_oracle.address().to_hopr_address(),
+            node_stake_v2_factory: instances.node_stake_v2_factory.address().to_hopr_address(),
```

Comment:
Beware that the node stake factory has different naming in hopr-bindings:

https://github.com/hoprnet/contracts/blob/95289351c46a0ffd0644728a47a279ade36549fb/ethereum/bindings/src/config.rs#L30